src/database/migrations/20221007124138-create-review-status.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) =>
+    queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        "review_requests", // name of Source model
+        "review_status", // name of column we're adding
+        {
+          type: Sequelize.ENUM,
+          values: ["OPEN", "MERGED", "CLOSED", "APPROVED"],
+          allowNull: false,
+          defaultValue: "OPEN",
+          transaction: t,
+        }
```

Comment:
We should also drop this in `src/tests/teardown.ts` as it is causing issues when running `npm run tests` repeatedly.

Reply:
fixed